src/main/java/net/minecraftforge/fml/common/eventhandler/EventBus.java

```diff
@@ -151,7 +151,7 @@ public void invoke(Event event)
             ArrayList<IEventListener> others = listeners.computeIfAbsent(target, k -> new ArrayList<>());
             others.add(listener);
         }
-        catch (Exception e)
+        catch (Throwable e)
```

Comment:
I have never seen something with `throws Throwable` before. I guess this is fine then.